tests/tests/integration_tests.rs

```diff
@@ -251,17 +251,15 @@ async fn http_making_more_requests_than_allowed_should_not_deadlock() {
 }
 
 #[tokio::test]
-#[ignore]
 async fn https_works() {
-	let client = HttpClientBuilder::default().build("https://kusama-rpc.polkadot.io").unwrap();
+	let client = HttpClientBuilder::default().build("https://kusama-rpc.polkadot.io:443").unwrap();
```

Comment:
isn't 443 assumed anyway when "https"?

Reply:
you are correct but [hyper::Uri](https://github.com/hyperium/http/issues/11) doesn't support default ports